routes/message.go

```diff
@@ -439,7 +440,7 @@ func (fes *APIServer) GetMessagesStateless(ww http.ResponseWriter, rr *http.Requ
 type SendMessageStatelessRequest struct {
 	SenderPublicKeyBase58Check    string `safeForLogging:"true"`
 	RecipientPublicKeyBase58Check string `safeForLogging:"true"`
-	MessageText                   string
+	EncryptedMessageText          string
```

Comment:
I'd like to maintain backwards compatibility for at least two weeks to give apps enough time to update. Let's keep `MessageText` and add `EncryptedMessageText`